src/sdk/pynni/nni/gridsearch_tuner/gridsearch_tuner.py

```diff
@@ -142,15 +194,24 @@ def generate_parameters(self, parameter_id, **kwargs):
         raise nni.NoMoreTrialError('no more parameters now.')
 
     def receive_trial_result(self, parameter_id, parameters, value, **kwargs):
+        """
+        Receive a trial's final performance result reported through ``nni.report_final_result`` by the trial.
```

Comment:
`report_final_result` should have a cross-reference.

Reply:
done